src/controller/python/chip-device-ctrl.py

```diff
@@ -43,7 +43,6 @@
 from xmlrpc.server import SimpleXMLRPCServer
 from enum import Enum
 from typing import Any, Dict,Optional
-
 # Extend sys.path with one or more directories, relative to the location of the
```

Comment:
This line removal looks like its outside our zone of operation?

Reply:
Resolved